tasks/configuration.yml

```diff
@@ -0,0 +1,93 @@
+---
+
+- name: "openstack_heat.tasks.configuration: Configure heat api"
+  ini_file:
+    dest: "/etc/heat/heat.conf"
+    section: "{{ item.section }}"
+    option: "{{ item.option }}"
+    value: "{{ item.value }}"
+  with_items:
+    - section:  database
```

Comment:
This alignment is good looking but does not comply with style and yamlint, we should get rid of it